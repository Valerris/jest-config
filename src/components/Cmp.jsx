import React, { useEffect } from "react"
import { ButtonS } from "components/styled"

async function f() {
	const rs = await fetch("/from-cmp")

	const res = await rs.json()

	console.log(res)
}

export default function Cmp() {
	useEffect(() => {
		// f()
	})

	return (
		<div>
			I'm a new Cmp.
			<ButtonS>Click</ButtonS>
		</div>
	)
}
