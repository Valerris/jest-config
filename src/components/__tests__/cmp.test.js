import React from "react"
import Cmp from "../Cmp"
import { shallow } from "enzyme"
import { expect } from "@jest/globals"

test("Cmp", () => {
	const wrapper = shallow(<Cmp />)

	expect(wrapper).toMatchSnapshot()
})
