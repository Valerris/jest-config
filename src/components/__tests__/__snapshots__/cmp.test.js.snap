// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cmp 1`] = `ShallowWrapper {}`;
